src/test/javascript/spec/component/text-editor/text-editor.component.spec.ts

```diff
@@ -97,6 +104,7 @@ describe('TextEditorComponent', () => {
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
                 { provide: TextSubmissionService, useClass: MockTextSubmissionService },
+                { provide: TranslateService, useClass: MockTranslateService },
             ],
         })
             .overrideModule(ArtemisTestModule, { set: { declarations: [], exports: [] } })
```

Comment:
same as above, what is this `overrideModule` for?

Reply:
and as well a comment to line 122 (which I am not able to annotate directly) `afterEach(() => ...` we could exchange it by `sinon.restore()`